apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -16,18 +16,35 @@
             <span class="input-group-text" id="pl-string-input-{{uuid}}-label">{{{label}}}</span>
         </span>
         {{/label}}
+        {{^multiline}}
         <input
+        {{/multiline}}
+        {{#multiline}}
+        <textarea
+            wrap="soft"
+        {{/multiline}}
             name="{{name}}"
             type="text"
             inputmode="text"
             class="form-control pl-string-input-input"
-            size="{{size}}"
+            {{^multiline}}
+                size="{{size}}"
+                {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
+            {{/multiline}}
+            {{#multiline}}
+                cols="{{size}}"
+                rows="1"
+            {{/multiline}}
             autocomplete="off"
             {{^editable}}disabled{{/editable}}
-            {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
             aria-describedby="pl-symbolic-input-{{uuid}}-label pl-symbolic-input-{{uuid}}-suffix"
             placeholder="{{placeholder}}"
-        />
+        {{^multiline}}
+        >
+        {{/multiline}}
+        {{#multiline}}
+        >{{#raw_submitted_answer}}{{raw_submitted_answer}}{{/raw_submitted_answer}}</textarea>
```

Comment:
I believe this is equivalent:
```suggestion
        >{{raw_submitted_answer}}</textarea>
```

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/f17253a7840c5847eeecaee0d49350fd4ef772b9